doc/man7/openssl_user_macros.pod.in

```diff
@@ -57,32 +52,39 @@ For convenience, higher numbers are accepted as well, as long as
 feasible.  For example, C<0x60000000L> will work as expected.
 However, it is recommended to start using the second form instead:
 
-=item C<m>
+=item C<mmnnpp>
+
+This form is a simple decimal number calculated with this formula:
+
+I<major> * 10000 + I<minor> * 100 + I<patch>
 
-This form is a simple number that represents the major version number
-and is supported for version 3.0 and up.  For extra convenience,
-these numbers are also available:
+where I<major>, I<minor> and I<patch> are the desired major,
+minor and patch numbers of the major number.  For example:
```

Comment:
This reads strange: ...desired major, minor and patch numbers of the *major* number?

Reply:
Yup. Typo, that "major" should be "version"

Reply:
Ping @mattcaswell for statement on the Oxford comma (adding comma in two places after minor)

Reply:
Fixed the *major* issue.  The Oxford comma is debatable in my view.